src/hotspot/share/gc/g1/g1BlockOffsetTable.hpp

```diff
@@ -51,9 +51,8 @@ class G1BlockOffsetTable: public CHeapObj<mtGC> {
   // The reserved region covered by the table.
   MemRegion _reserved;
 
-  // Array for keeping offsets for retrieving object start fast given an
-  // address.
-  volatile u_char* _offset_array;  // byte array keeping backwards offsets
+  // Biased array-start of BOT array for fast BOT entry translation
+  volatile u_char* _offset_base;
```

Comment:
Maybe the type could be changed to `uint8_t` as well while we are changing this code but feel free to ignore at this point.

Reply:
Filed https://bugs.openjdk.org/browse/JDK-8329096 to follow up.